### PR TITLE
Update API documentation

### PR DESCRIPTION
**Pull request by TheYoBots**
Please do mention if there are any mistakes.

### CODE REVIEW

File: public/doc/lishogi-api.yaml [hunk 1]
```diff
@@ -4405,2267 +3153,236 @@ paths:
...
-              schema:
-                $ref: '#/components/schemas/Ok'
-        "400":
-          description: The victory claim has failed.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-
-  /api/bot/online:
```
Comment (ddugovic) [inline public/doc/lishogi-api.yaml:4825]: Please see #580

Comment (TheYoBots) [reply public/doc/lishogi-api.yaml:4825]: https://github.com/lichess-org/lila/commit/4df218c7ede4431cf30f9b8c7b78c1cb391cc1c5 Isn't this the lichess commit? Does PR add the endpoint `/api/bot/online` or it directs to some other endpoint?

Comment (ddugovic) [reply public/doc/lishogi-api.yaml:4825]: Yes, this PR adds the GET /api/bot endpoint:
```
GET   /api/bot/*cmd                    controllers.PlayApi.botCommandGet(cmd: String)
```

Comment (TheYoBots) [reply public/doc/lishogi-api.yaml:4825]: https://github.com/TheYoBots/lishogi/blob/lishogi-api-doc/public/doc/lishogi-api.yaml#L2790 I hadn't removed the endpoint that you added in your PR.

Comment (ddugovic) [reply public/doc/lishogi-api.yaml:4825]: My PR adds all `GET /api/bot/*` paths, I think; previously there were none.

Comment (TheYoBots) [reply public/doc/lishogi-api.yaml:4825]: As per what I see in your PR here https://github.com/WandererXII/lishogi/pull/580/files all changes has something to do with chat. The name of the commit too is to add the endpoint `GET /api/bot/game/{gameId}/chat`.

Incase you want to add the endpoint `GET api/bot/online` then the lichess commit you are looking for is this https://github.com/lichess-org/lila/commit/4df218c7ede4431cf30f9b8c7b78c1cb391cc1c5

Comment (ddugovic) [reply public/doc/lishogi-api.yaml:4825]: Thanks, you are correct.  I've fixed my PR and my test site http://dugovic.mooo.com/ is running, although I do not know how to test a Lishogi BOT (can connect and play on it).

**Comment by WandererXII** [comment]
Looks great, thanks for going through that, must have taken a lot of time!

**Pull request merged**